WordPress/Classes/Services/JetpackNotificationMigrationService.swift

```diff
@@ -19,10 +28,15 @@ final class NotificationFilteringService {
                 return true
             }
 
-            return userDefaults.bool(forKey: WPNotificationsEnabledKey)
+            return userDefaults.bool(forKey: WPNotificationsEnabledKey) && UIApplication.shared.isRegisteredForRemoteNotifications
         }
 
         set {
+            if newValue, AppConfiguration.isWordPress {
```

Comment:
```suggestion
            if newValue, isWordPress {
```

Could the local variable be used which is populated from `AppConfiguration.isWordPress` (except in tests where it's injected)?

Reply:
Oh yes, the local variable should be used here 👍